src/EcommerceDDD.Spa/src/app/shared/components/stored-events-viewer/stored-events-viewer.component.ts

```diff
@@ -7,9 +7,9 @@ import { StoredEventData } from '@ecommerce/models/StoredEventData';
   styleUrls: ['./stored-events-viewer.component.scss'],
 })
 export class StoredEventsViewerComponent {
-  @Output('destroyComponent') destroyComponent: EventEmitter<any> =
-    new EventEmitter();
-  storedEventData!: StoredEventData[];
+  @Output('destroyComponent') destroyComponent: EventEmitter<void> =
+    new EventEmitter(); // this can rather be reworked for simplification and less coupling
```

Comment:
I recommend you place a // TODO: instead of a simple //, so we can keep track of it

Reply:
Updated  🤙🏻